content/docs/command-reference/remote/add.md

```diff
@@ -47,13 +47,6 @@ DVC will determine the [type of remote](#supported-storage-types) based on the
 > include them all. The command should look like this: `pip install "dvc[s3]"`.
 > (This example installs `boto3` library along with DVC to support S3 storage.)
 
-DVC supports the concept of a _default remote_. For the commands that accept a
-`--remote` option (`dvc pull`, `dvc push`, `dvc status`, `dvc gc`, `dvc fetch`),
-the default remote is used if that option is not used.
```

Comment:
OK this part I'm not sure I should've removed indeed... ⌛

Reply:
I think it should go in the remote cmd ref index though. I rewrote that text and included this note. See e8713864 please @shcheklein.

Reply:
I think it's import to keep it here (not sure about index). Moreover I would even move up and highlight. Note: if you add the first remote you most likely want to use `-d` option as well. 

It's important since this is most likely the command people would to use.

Reply:
I agree. I was just assuming people would spot to read the index first but now that you mention it, yeah probably a lot of people will only skim over it the first time or skip it altogether on their way to the `remote add` ref.

OK, I rolled back some of the changes to the index and emphasized on the default remote in add. See https://github.com/iterative/dvc.org/compare/ab13a8c..e657efd for a summary of the changes related to this matter.

Reply:
I guess since this was the only concern and I've addressed it, I should merge this now 🎉 Thanks